cli/src/build/onboarding/ui/steps/android-keystore.tsx

```
@@ -0,0 +1,363 @@
+
+export interface KeystoreExplainerStepProps {
+  onBack: () => void
+  dense?: boolean
+}
+
+// Comfortable: the original info Alert + a <Newline/> + four indented,
+// full-sentence bullets in a marginLeft box + a <Newline/> + the Back control.
+// Dense: the Alert/box/blank-lines are dropped in favour of terse single-line
+// bullets so every line stays un-wrapped within the 13-row budget at 60 cols
+// (the original wrapping bullets blew the budget there).
+export const KeystoreExplainerStep: FC<KeystoreExplainerStepProps> = ({ onBack, dense = false }) => {
```

ESLint

[error] 'dense' is assigned a value but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)